invoice.py

```diff
@@ -207,6 +213,21 @@ def payment_details(self):
                 if ml.account.type.receivable],
             key=attrgetter('maturity_date'))
 
+    @classmethod
+    def post(cls, invoices):
+        Config = Pool().get('account.configuration')
+
+        config = Config(1)
+        transaction = Transaction()
+        context = transaction.context
+
+        super(Invoice, cls).post(invoices)
+
+        with transaction.set_context(
+                queue_scheduled_at=config.invoice_facturae_after,
+                queue_batch=context.get('queue_batch', True)):
```

Comment:
What is queue_batch useful for?

Reply:
for grouped_slice count:

https://github.com/tryton/tryton/blob/5366e00f561ecf273b08e9d25884f0b5d76a1c9c/trytond/trytond/ir/queue_.py#L249

There are some module use "queue_batch":

```
modules/purchase/purchase.py
modules/sale/sale.py
modules/account_invoice/invoice.py
....
```